modules/appmodel/app-parser-generator.c

```diff
@@ -175,7 +194,7 @@ _generate_framing(AppParserGenerator *self, GlobalConfig *cfg)
         g_string_append(self->block, "        channel {\n");
 
       g_string_append(self->block,
-                      "            filter { tags('.app.doesnotexist'); };\n"
+                      "            filterx { false; };\n"
```

Comment:
`_generate_action()` can still generate "traditional" `rewrite`, too.

I think we should either support both everywhere or support only filterx.
This is true for `_generate_empty_frame()`, too.

What do you think?

Reply:
Talked about this IRL. Functionally these 2 are the same, but `filterx` is quicker if we ever used a `filterx` block, as the simple `filter` block would force a `LogMessage` sync.